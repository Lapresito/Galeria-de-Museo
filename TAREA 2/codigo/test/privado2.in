crearVisitante 100 Juan Martinez 30
imprimirVisitante
imprimirEdadVisitante
imprimirIdVisitante
copiarVisitante
imprimirVisitante
liberarVisitante
Fin