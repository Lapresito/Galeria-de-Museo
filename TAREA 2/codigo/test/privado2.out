Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Visitante creadox de forma exitosa.
2> Visitante Juan Martinez
Id: 100
Edad: 30
3> La edad del visitante es: 30
4> La id del visitante es: 100
5> Visitante copiado. Datos de la copia:
Visitante Juan Martinez
Id: 100
Edad: 30
6> Visitante Juan Martinez
Id: 100
Edad: 30
7> Visitante liberado con exito.
8> Fin.
